app/assets/components/messages/ember-cli-build.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
This is only useful if we run `ember server` and modify the component in isolation. I don't think this will be possible unless we integrate our socket javascript into the component itself …